codespell_lib/data/dictionary.txt

```diff
@@ -24778,7 +24917,9 @@ nubmers->numbers
 nucleous->nucleus, nucleolus,
 nucular->nuclear
 nuculear->nuclear
+nuisans->nuisance
 nuisanse->nuisance
+nuisanses->nuisance
```

Comment:
`nuisanses->nuisance, nuisances,`
I haven't ever encountered the plural form `nuisances`, but it can be found in [Collins](https://www.collinsdictionary.com/dictionary/english/nuisance) for example.

Reply:
Yup, added.